README.md

```diff
@@ -94,4 +94,4 @@ and [Sprig](https://github.com/Masterminds/sprig) library.
 
 ### Built-in functions
 There are a few built in functions as well:
-  * `require (env "ENV_NAME")` - Renders an error if environments variable does not exists. If it is equal to empty string, returns empty string.  `{{ require (env "TIMEOUT_MS) }}`
+  * `require (env "ENV_NAME")` - Renders an error if environments variable does not exists or it is equal to empty string. `{{ require (env "TIMEOUT_MS) }}`
```

Comment:
It is not the subject of your change but I think that "_environments variable_" should be replaced with "_the environment variable_".

Reply:
Fixed in 7fd4159, take a look